scripts/release.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -e
+
+RELEASE_TYPE="$1"
+if [ -z "$RELEASE_TYPE" ]
+then
+ RELEASE_TYPE="release"
+fi
+
+if [[ $RELEASE_TYPE != "release" ]] && [[ $RELEASE_TYPE != "release" ]]
+then
+  echo "release or prerelease are the only acceptable options"
+  exit 1
+fi
+
+# check gh auth
+if ! gh auth status&> /dev/null; then
+    gh auth login
+fi
+
+DIR="$( cd -P "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# call bumper
+yarn run bump "$RELEASE_TYPE"
+
+# get last tag
+LAST_TAG=$(git describe --tags --abbrev=0)
+
+# generate changelog
+yarn run changelog $LAST_TAG
+
+# get current version
+PACKAGE_VERSION=$(cat $DIR/../package.json|grep version|head -1|awk -F: '{ print $2 }'|sed 's/[", ]//g')
+
+if [[ $RELEASE_TYPE == "release" ]]
+then
+ BRANCH="master"
+else
+ BRANCH=$(git branch --show-current)
```

Comment:
We probably need to get branch from version

Reply:
It would be exactly the same

Reply:
But what if we are bumping not from a dev branch?

Reply:
Agreed to check that the current branch and base branch are the same, otherwise we exit with an error